chsdi/templates/htmlpopup/astra_schwerverunf_kanton_alkohol.mako

```diff
@@ -0,0 +1,19 @@
+<%inherit file="base.mako"/>
+<%def name="table_body(c, lang)">
+<%
+   import locale
+   locale.setlocale(locale.LC_ALL, 'fr_CH.utf-8')
```

Comment:
@loicgasser I've no idea... it's not my code ;)
I'll remove that if you say it's safe to do so